plugins/splitwindow.c

```diff
@@ -318,10 +318,15 @@ static void split_view(gboolean horizontal)
 	gtk_container_add(GTK_CONTAINER(pane), notebook);
 	g_object_unref(notebook);
 
+	/* used just to get us to the same visual height level as the main editor window */
+	viewport = gtk_viewport_new(NULL, NULL);
```

Comment:
Hum, I just tested all the themes I have installed, and I'm afraid it's just not possible to fake the appearance of e.g. a ScrolledWindow with anything but a ScrolledWindow.  Some theme style ScrolledWindow and Viewport the same and Frame differently, some style ScrolledWindow and Frame the same but Viewport different, etc. -- basically all combinations you could think of.

As IIUC here the goal is to look like a GtkNotebook, I think the only effective way would be to actually use a GtkNotebook.  It might be a bit overkill, but a notebook with the tabs hidden and only one page should do it just fine.